csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -31,6 +31,19 @@ public enum ExecutionMode
         ORT_PARALLEL = 1,
     }
 
+    /// <summary>
+    /// Language projection property for telemetry event for tracking the source usage of ONNXRUNTIME
+    /// </summary>
+    public enum OrtProgrammingProjection
```

Comment:
does this need to be here and in the c_api header? can it just be in one spot?

Reply:
Similar as ExecutionMode and GraphOptimizationLevel, we need to have copies of this for C#, because it doesn't have direct access to c_api header.